File: buffalo_bench/models/order.go
package models

import (
	"time"
)

type Order struct {
	ID        int64     `db:"id"`
	OrderCode string    `db:"order_code"`
	Division  string    `db:"division"`
	CreatedAt time.Time `db:"created_at"`
	UpdatedAt time.Time `db:"updated_at"`
}

File: octane/app/Models/Order.php
<?php

namespace App\Models;

use Carbon\CarbonImmutable;
use Illuminate\Database\Eloquent\Model;

/**
 * @property int $id
 * @property string $division
 * @property string $order_code
 * @property CarbonImmutable $created_at
 * @property CarbonImmutable $updated_at
 */
class Order extends Model
{
    public static $snakeAttributes = false;

    protected $hidden = ['created_at', 'updated_at'];

}
